src/microbe_stage/MicrobeAISystem.cs

```diff
@@ -76,8 +76,11 @@ private void RunAIFor(IMicrobeAI? ai, float delta, Random random, MicrobeAICommo
         if (ai.TimeUntilNextAIUpdate > 0)
             return;
 
+        // TODO: would be nice to add a tiny bit of randomness to the times here so that not all cells think at once
         ai.TimeUntilNextAIUpdate = Constants.MICROBE_AI_THINK_INTERVAL;
 
-        ai.AIThink(delta, random, data);
+        // As the AI think interval is made constant, we pass that value as the delta to make time keeping be actually
+        // (mostly) consistent in the AI code
+        ai.AIThink(Constants.MICROBE_AI_THINK_INTERVAL, random, data);
```

Comment:
I'm unsure that this will be accurate. We only have assurance that time since last interval is at least MICROBE_AI_THINK_INTERVAL, but it could have been significantly more, meaning an accurate time delta would be higher. I don't think that the AI actually uses that delta yet, but if we're passing it in it should be accurate.

Reply:
It does now for my signaling agents calculation.
I just made it better. Before it passed the delta for the *previous* frame. Which is way, way more incorrect here. I realize my solution is not perfect, and I did write that this is mostly right. As long as we don't have massive lag spikes the time elapsed should be consistent enough.